<?xml version="1.0"?>
<launch>

   <node name="view_fsm" pkg="smach_viewer" type ="smach_viewer.py" />
   <node name="oracle" pkg="assignment1" type ="oracle.py"/>
   <node name="state_machine"  pkg="assignment1" type="state_machine.py" output="screen"/>
   <node name="go_to_point"  pkg="assignment1" type="go_to_point.py" />

   

</launch>
